src/test/java/com/segment/analytics/android/integrations/comscore/ComScoreTest.java

```diff
@@ -637,53 +596,43 @@ public void videoContentPlayingWithAdType() {
     expected.put("c4", "*null");
     expected.put("c6", "*null");
 
-    Mockito.verify(playbackSession).setAsset(expected);
-    Mockito.verify(streamingAnalytics).notifyPlay(70);
+    Mockito.verify(streamingAnalytics).setMetadata(refEq(getAdvertisementMetadata(expected)));
```

Comment:
@breel93  Replacing this line with `Mockito.verify(streamingAnalytics, atLeast(1)).setMetadata(refEq(getContentMetadata(expected)));` should fix the failing test. Here if I am understanding correctly we are expecting content metadata (as we are tracking a return to content from an ad break) rather than ad metadata.

Reply:
Yes, I totally missed that
I am on it thanks so much